plugin/yt_dlp_plugins/extractor/getpot_bgutil_http.py

```diff
@@ -91,7 +91,11 @@ def _check_server_availability(self, ctx: PoTokenRequest):
                 f'Unknown error reaching GET /ping (caused by {e!r})', raise_from=e)
             return
         else:
-            self._check_version(response.get('version'), name='HTTP server')
+            server_version = response.get('version', '')
+            self._check_version(server_version, name='HTTP server')
+            if not server_version or self.PROVIDER_VERSION.split('.', 1)[0] != server_version.split('.', 1)[0]:
+                self._warn_and_raise(
+                    'Mismatched major versions are not expected to work together.')
```

Comment:
Just want to reword this a little bit to have a call to action as well. `Plugin and server major versions are mismatched. Update both the plugin/server to the same version to proceed.`

Reply:
Done.